server.go

```
@@ -57,6 +58,10 @@ var DuplicatePeerError = errors.New("raft.Server: Duplicate peer")
 var CommandTimeoutError = errors.New("raft: Command timeout")
 var StopError = errors.New("raft: Has been stopped")
 
+// ErrSnapshotInProgress is returned when a snapshot is asked for while another
+// one is still being written.
+var ErrSnapshotInProgress = errors.New("Snapshot: Last snapshot is not finished.")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not end with punctuation or newlines

(staticcheck)